articles/faq/add-Ip-to-existing-paid-plan.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Can I add a Dedicated IP to my existing Paid plan?"
+description: "The older paid plans no longer offer the ability to add dedicated IPs to your account. If you require Dedicated IPs please upgrade to one of our new plans from the Accounts/Billing page within the SparkPost UI (https://app.sparkpost.com/account/billing).
+Please note that new plans from 150K upwards include a dedicated IP, allowing you to add an IP Pool and new IP. You can of course add multiple IPs at 20USD per month in that or other IP Pools you create."
+---
+
+The older paid plans no longer offer the ability to add dedicated IPs to your account. If you require Dedicated IPs please upgrade to one of our new plans from the Accounts/Billing page within the SparkPost UI (https://app.sparkpost.com/account/billing).
```

Comment:
Remove `...within the SparkPost UI` and make Accounts/Billing page a link like this:

`[Accounts/Billing page](https://app.sparkpost.com/account/billing)`